myTrip/like/models.py

```diff
@@ -0,0 +1,117 @@
+"""This module contains like model class and basic functions."""
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import models
+
+
+class Like(models.Model):
+    """
+    Like
+    :argument id: int - auto generate primary key
+    :argument user: int - ToDo foreign key
+    :argument trip: int - ToDo foreign key
+    :argument checkpoint: int - ToDo foreign key
+    :argument photo: int - ToDo foreign key
+    :argument comment: int - ToDo foreign key
+    """
+    user = models.IntegerField()
+    trip = models.IntegerField(null=True)
+    checkpoint = models.IntegerField(null=True)
+    photo = models.IntegerField(null=True)
+    comment = models.IntegerField(null=True)
+
+    @staticmethod
+    def get_by_id(like_id):
+        """
+        ToDo method to get one element by his id, uses to views,
```

Comment:
Remove 'ToDo's from everywhere, when pushing to remote repo.
Gonna be like this: A method to get specific element by id. Used in views. Returns None when exception works.